keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -242,12 +557,12 @@ _This section must be completed when targeting beta graduation to a release._
 * **How can an operator determine if the feature is in use by workloads?**
 
 * Check Service to see if `internalTrafficPolicy` is set to `Local`.
-* A per-node "blackhole" metric will be added to kube-proxy which represent Services that are being intentionally dropped (internalTrafficPolicy=Local and no endpoints). The metric will be named `kubeproxy/sync_proxy_rules/blackhole_total` (subject to rename).
+* A per-node "blackhole" metric will be added to kube-proxy which represent Services that are being intentionally dropped (internalTrafficPolicy=Local and no endpoints). The metric will be named `kubeproxy/sync_proxy_rules_no_endpoints_total` (subject to rename).
```

Comment:
I think it's `sync_proxy_rules_no_local_endpoints_total` instead?
If so, can you please update (and remove "subject to rename :) )

Reply:
Also L570: I think this metrics was already added, right?

Reply:
Thanks, updated